lib/node_modules/@stdlib/stats/base/dists/triangular/logpdf/benchmark/benchmark.js

```diff
@@ -45,12 +45,11 @@ bench( pkg, function benchmark( b ) {
 	min = new Float64Array( len );
 	max = new Float64Array( len );
 	mode = new Float64Array( len );
-
 	for ( i = 0; i < len; i++ ) {
-		x[ i ] = randu() * 30.0;
-		min[ i ] = randu() * 10.0;
-		max[ i ] = min[ i ] + ( randu() * 40.0 ) + EPS;
-		mode[ i ] = min[ i ] + ( ( max[ i ] - min[ i ] ) * randu() );
+		x[ i ] = uniform( 0.0, 30.0 );
+		min[ i ] = uniform( 0.0, 10.0 );
+		max[ i ] = uniform( min[ i ] + EPS, 40.0+min[ i ] + EPS );
```

Comment:
```suggestion
		max[ i ] = uniform( min[ i ] + EPS, 40.0+ EPS + min[ i ] );
```

Be consistent with `native.js`.